client/src/ui/components/GameSelection.tsx

```diff
@@ -1,57 +1,56 @@
-import { SetStateAction, useState } from "react";
+import { useState } from "react";
 import Dropdown from "./Dropdown";
+
 const GameSelection = () => {
-  const [selectedOption, setSelectedOption] = useState("");
+  const [selectedOption, setSelectedOption] = useState<string[]>([]);
 
-  const handleSelectionChange = (selectedValue: SetStateAction<string>) => {
-    console.log("New dropdown selection: ", selectedValue);
-    setSelectedOption(selectedValue);
+  const handleSelectionChange = (selectedValues: string[]) => {
+    console.log("New dropdown selection: ", selectedValues);
+    setSelectedOption(selectedValues);
   };
 
-  // DropdownList will return the required list for the dropdown with SQL
-  const list = () => {
-    return [
-      "Valorant",
-      "League of Legends",
-      "Clash of Clans",
-      "Lethal Company",
-      "Apex",
-    ];
-  };
+  const list = [
+    "Valorant",
+    "League of Legends",
+    "Clash of Clans",
+    "Lethal Company",
+    "Apex",
+    "Fortnite",
+  ];
+
+  const selectedList: string[] = [];
 
-  // Back-end after a game has been chosen from the dropdown
   const handleFindClick = () => {
-    console.log("Finding", selectedOption, "lobbies.");
+    console.log("Finding lobbies for:", selectedOption.join(", "));
```

Comment:
Got rid of the other console.log, keeping this one because typescript will give me a variable not used error. Adding TODO comment since this function needs to query database.